src/Dom.ts

```diff
@@ -404,6 +409,7 @@ export class HTMLForm extends VisualForm {
     this._space.add( { start: () => {
       this._ctx.group = this._space.element;
       this._ctx.groupID = "pts_dom_"+(HTMLForm.groupID++);
+      this._ctx.style = this._style;
```

Comment:
Should this use `Object.assign({}, this._style)` instead?

Reply:
You're right 🤦‍♂️